src/groups/mqb/mqbc/mqbc_clusterutil.cpp

```diff
@@ -1390,9 +1390,18 @@ void ClusterUtil::updateAppIds(ClusterData*                    clusterData,
                        << queueAdvisory << ", rc: " << rc;
     }
     else {
-        BALL_LOG_INFO << "Unregister appIds " << printRemoved
-                      << " and registered appIds " << printAdded << " for '"
-                      << uri << "'.";
+        BALL_LOG_INFO_BLOCK
+        {
+            BALL_LOG_OUTPUT_STREAM << "Advisory applied: unregister appIds "
+                                   << printRemoved << " and register appIds "
+                                   << printAdded << " for ";
+            if (!uri.empty()) {
+                BALL_LOG_OUTPUT_STREAM << " uri = [" << uri << "]";
+            }
+            if (!domainName.empty()) {
```

Comment:
This should always be available/non-empty. Looks like `uri` is the only optional one (not a big fan of this API, but that's a separate discussion).